src/server/api/schema.js

```diff
@@ -444,6 +444,26 @@ const rootMutations = {
       }
       return organization
     },
+    assignUserToCampaign: async (_, { campaignId }, { user, loaders }) => {
+      let campaign
+      [campaign] = await r.knex('campaign')
+        .where('id', campaignId)
+      if (campaign) {
+        const assignment = await r.table('assignment')
+          .getAll(user.id, { index: 'user_id' })
+          .filter({ campaign_id: campaign.id })
+          .limit(1)(0)
+          .default(null)
+        if (!assignment) {
+          await Assignment.save({
+            user_id: user.id,
+            campaign_id: campaign.id,
+            max_contacts: (process.env.DEFAULT_MAX_CONTACTS || 1)
```

Comment:
It looks like `max_contacts` is a new field which we either need to add in this PR, or we can remove this line and include in a max_contacts PR separately.